.travis.yml

```diff
@@ -13,13 +13,14 @@ services:
   - docker
 
 env:
-  - DEX_POSTGRES_DATABASE=postgres DEX_POSTGRES_USER=postgres DEX_POSTGRES_HOST="localhost" DEX_ETCD_ENDPOINTS=http://localhost:2379 DEX_LDAP_TESTS=1 DEBIAN_FRONTEND=noninteractive
+  - DEX_POSTGRES_DATABASE=postgres DEX_POSTGRES_USER=postgres DEX_POSTGRES_HOST="localhost" DEX_ETCD_ENDPOINTS=http://localhost:2379 DEX_LDAP_TESTS=1 DEBIAN_FRONTEND=noninteractive DEX_KEYSTONE_URL=http://localhost:5000 DEX_KEYSTONE_ADMIN_URL=http://localhost:35357
 
 install:
   - sudo -E apt-get install -y --force-yes slapd time ldap-utils
   - sudo /etc/init.d/slapd stop
   - docker run -d --net=host gcr.io/etcd-development/etcd:v3.2.9
-
+  - docker run -d -p 0.0.0.0:5000:5000 -p 0.0.0.0:35357:35357 openio/openstack-keystone
+  - sleep 60s
```

Comment:
🤔 Can we do something that is a bit more robust? 
Maybe (untested)
```yaml
  - |
    until curl --fail http://127.0.0.1:5000/v3; do
      echo 'Waiting for keystone...'
      sleep 1;
    done;
```
This would potentially be faster (wait for less than 60s) -- OTOH, it could take longer (if keystone does not come up), but I'd hope travis will fail the build with some timeout in that case.

Either way, this new additional dependency in our test suite is something to keep an eye on when this is merged. 🔍